package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "eth2-crawler-ui",
+  "name": "ethereum-consensus-crawler-ui",
```

Comment:
I'm imagining it's okay to change the package name since this isn't published to NPM

Reply:
[This](https://github.com/ChainSafe/nodewatch-ui/pull/59/commits/12775ed72b780557d29033be8b415ab824ef31ee) commit renames the package to `nodewatch-crawler-ui`